src/CAN/can_motor.cpp

```diff
@@ -0,0 +1,44 @@
+#include "can_motor.h"
+#include "../world_interface/real_world_constants.h"
+#include "CAN.h"
+#include "CANMotor.h"
+#include "CANUtils.h"
+
+namespace robot {
+class can_motor: public base_motor {
+	public: 
```

Comment:
Check the style guide, you should be able to automate this with your editor. `public`/`private` directive aren't indented, and the body of a class is indented by one.

Reply:
Done